lib/scarpe/para.rb

```diff
@@ -1,13 +1,31 @@
 module Scarpe
   class Para
-    def initialize(app, text)
+    def initialize(app, text, options = {})
       @app = app
       @text = Array(text)
+      @options = Hash(options)
       @app.append(render)
     end
 
     def render
-      "<p id=#{object_id}>#{text.join}</p>"
+      HTML.render do |h|
+        h.p(attributes) do
+          @text.join
+        end
+      end
+    end
+
+    def attributes
+      {
+        id: object_id,
+        style: (style if style.any?)
```

Comment:
I think we can move this logic to the HTML module
as in you can put in `style: {}` and it would not render them

Reply:
There was a minor bug that an empty hash would still cause a `style=""` to be output, but I think I've fixed that in here now.